src/App.css

```diff
@@ -1,43 +1,32 @@
 .App {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin: 0 auto;
   text-align: center;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.FAQ {
```

Comment:
:wrench: Everything related to the FAQ page needs to be moved to a separate PR. We want to keep our PR's small and for one thing at a time.